### DISCUSSION TITLE
Pin branch gets additionally created when grouped with other

### DISCUSSION THREAD

**Discussion by mihainsto** [renovatebot/renovate#9423] [Request Help] [answered]
Hi, so I want to setup the renovate bot to produce only two branches one for non-major updates and one for major updates, the thing is that I got this additional branch Pin and I don't want that, I want it to be with non-major updates.

Here is my config:
```
"packageRules": [
    {
      "matchUpdateTypes": ["patch", "minor", "pin"],
      "groupName": "dependencies (non-major)"
    },
    {
      "matchUpdateTypes": ["major"],
      "groupName": "dependencies (MAJOR)"
    }
  ]
```

The result produces by renovate:
![image](https://user-images.githubusercontent.com/44275480/113833305-2f543800-9792-11eb-8dd2-7ca09e2cd6ca.png)

Any ideas? Thanks!

**Comment by rarkins** [comment, marked as answer]
Pinning dependencies is a one-time process and is put into its own group intentionally. i.e. what you are seeing is by design

You should either:
- Merge the Pin dependencies PR first, and after that be left with the two groups you intend, or
- Pin dependencies manually before onboarding Renovate, or
- Reconfigure Renovate so that no pinning is done

**Reply by mihainsto** [reply]
Thank you, I will read more about pinning in the docs.
Have a nice day!
